spec/suma/i18n_spec.rb

```diff
@@ -58,6 +58,36 @@
     end
   end
 
+  describe "ensure_english_interpolation_values" do
+    it "ensures dynamic values are not changed" do
+      spanish_dynamic_str = "Precio es {{precio}}."
+      english_dynamic_str = "Price is {{price}}."
+      expect(described_class.ensure_english_interpolation_values(
+               spanish_dynamic_str,
+               english_dynamic_str,
+               "English",
+             )).to eq("Precio es {{price}}.")
+    end
+    it "ensures multiple dynamic values are not changed" do
+      spanish_dynamic_str = "Precio es {{precio}}. Salvaste {{cantidad}}."
+      english_dynamic_str = "Price is {{price}}. You saved {{amount}}."
```

Comment:
What happens if the ordering of translated strings is reversed? Ie, `{{price}} {{amount}}` and `{{cantidad}} {{precio}}`.

Reply:
Similarly, what happens for `{{price}}` and `{{precio cantidad}}` and `{{price}} {{amount}}` and `{{precio}}`?

Reply:
And add a test to make sure you don't get tripped up by whitespace within the braces.

Reply:
Test would fail due to reversed dynamic values.

Currently, we are only checking for languages that are not English (base_language). This works with our current CSV format so far but maybe there is a better way of checking.